spreadsheets/sort_rank/landuse.csv

```diff
@@ -18,85 +19,97 @@ groyne,*,225
 breakwater,*,224
 dam,*,223
 land,*,222
-footway,*,112
-tower,*,111
-wilderness_hut,*,110
-parking,*,109
-picnic_site,*,108
-water_park,*,107
-amusement_ride,*,106
-water_slide,*,105
-carousel,*,104
-attraction,*,103
-artwork,*,102
-animal,*,101
-pitch,*,100
-playground,*,99
-pedestrian,*,98
-dog_park,*,97
-garden,*,96
-beach,*,95
-hanami,*,94
-maze,*,93
-camp_site,*,92
-generator,*,91
-allotments,*,90
-plant,*,89
-summer_toboggan,*,88
-roller_coaster,*,87
-enclosure,*,86
-petting_zoo,*,85
-aviary,*,84
-theatre,*,83
-library,*,82
-cinema,*,81
-fuel,*,80
-school,*,79
-cemetery,*,78
-place_of_worship,*,77
-quarry,*,76
-railway,*,75
-farmyard,*,74
-substation,*,73
-works,*,72
-water_works,*,71
-wastewater_plant,*,70
-common,*,69
-village_green,*,68
-sports_centre,*,67
-stadium,*,66
-runway,*,65
-apron,*,64
-taxiway,*,63
-aerodrome,*,60
-recreation_track,*,59
-caravan_site,*,58
-recreation_ground,*,57
-golf_course,*,56
-aquarium,*,55
-hospital,*,54
-trail_riding_station,*,53
-resort,*,52
-zoo,*,51
-winery,*,50
-wildlife_park,*,49
-theme_park,*,48
-fort,*,47
-prison,*,46
-military,*,45
-retail,*,44
-commercial,*,43
-service_area,*,42
-rest_area,*,41
-college,*,40
-university,*,39
-farmland,*,38
-farm,*,37
-winter_sports,*,36
-wetland,*,35
-scrub,*,34
-meadow,*,33
-grass,*,32
+wetland,*,220
+footway,*,121
+tower,*,120
+wilderness_hut,*,119
+parking,*,118
+picnic_site,*,117
+water_park,*,116
+amusement_ride,*,115
+water_slide,*,114
+carousel,*,113
+attraction,*,112
+artwork,*,111
+animal,*,110
+pitch,*,109
+playground,*,108
+pedestrian,*,107
+dog_park,*,106
+garden,*,105
+beach,*,104
+hanami,*,103
+maze,*,102
+camp_site,*,101
+generator,*,100
+allotments,*,99
+plant,*,98
+summer_toboggan,*,97
+roller_coaster,*,96
+enclosure,*,95
+petting_zoo,*,94
+aviary,*,93
+theatre,*,92
+library,*,91
+cinema,*,90
+fuel,*,89
+school,*,88
+cemetery,*,87
+place_of_worship,*,86
+quarry,*,85
+railway,*,84
+farmyard,*,83
+substation,*,82
+works,*,81
+water_works,*,80
+wastewater_plant,*,79
+common,*,78
+village_green,*,77
+sports_centre,*,76
+stadium,*,75
+runway,*,74
+apron,*,73
+taxiway,*,72
+aerodrome,*,70
+airfield,*,69
+recreation_track,*,68
+caravan_site,*,67
+recreation_ground,*,66
+quay,*,65
+wharf,*,65
+port_terminal,*,64
+ferry_terminal,*,64
```

Comment:
Can't share sort values otherwise it creates flickr no Tangram ES... so this and quay and possibly others need to offset away from each other.

Reply:
I've re-arranged them in ffe23bd to make them unique. Also added a test, so that we'll get a test failure if a duplicate is added in future.